util/typeutil/stringInt_test.go

```diff
@@ -0,0 +1,28 @@
+package typeutil
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUnmarshalJSON(t *testing.T) {
+	type Item struct {
+		ItemId StringInt `json:"item_id"`
+	}
+
+	t.Run("happy path - convert string to int", func(t *testing.T) {
```

Comment:
Test names cannot have spaces, they are converted to underscored under the covers. Therefore, I recommend to use a simple name using hyphens or underscores, ex: `string` and `int` would be descriptive enough names IMHO given the context.

Reply:
this is done - https://github.com/prebid/prebid-server/commit/9e4089964012932b327478fc68b37b411a51bdcf